manifests/o/Overwolf/CurseForge/0.201.1-4729.4729/Overwolf.CurseForge.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Overwolf.CurseForge
+PackageVersion: 0.201.1-4729.4729

```

```suggestion
PackageVersion: 0.201.1-4729
```
This must match Control Panel.